variables.tf

```diff
@@ -103,6 +103,12 @@ variable "iam_service_linked_role" {
   description = "Whether or not to create aws_iam_service_linked_role. Defaults to `false`."
 }
 
+variable "kms_key_id" {
```

Comment:
Do you mind naming this `secretsmanager_kms_key_id` or something identifying that the key will be used for the created Secrets Manager secret?

Reply:
I have updated the variable name to `secretsmanager_kms_key_id`